lib/src/RadioButton.js

```diff
@@ -108,5 +122,8 @@ RadioButton.defaultProps = {
   onPress: () => null,
   text: "Radio Button",
   innerColor: "dodgerblue",
-  outerColor: "dodgerblue"
+  outerColor: "dodgerblue",
+  width: 35,
+  height: 35,
+  imageSource: require("../assets/images/chick.png"),
```

Comment:
Please remove the default one, also remove the whole `assets` folder inside `lib`. Library itself should not contain any images or assets. It should come from the developer.